runtime/src/main/scala/akka/grpc/internal/AkkaNettyGrpcClientGraphStage.scala

```diff
@@ -172,7 +172,7 @@ private final class AkkaNettyGrpcClientGraphStage[I, O](
           call = null
         }
         if (!matVal.isCompleted) {
-          matVal.failure(new AbruptStageTerminationException(this))
+          matVal.tryFailure(new AbruptStageTerminationException(this))
```

Comment:
I guess we can remove the `matVal.isCompleted` check now, then?

Reply:
Hmm.. 🤔 
Does it make sense to try if we know it's completed?

On the other hand removing the check would reduce the LOC, so I can do that if you want. 👍 
Should I?

Reply:
A matter of taste I suppose - let's leave it like this :+1: